stanford_help.install

```diff
@@ -0,0 +1,61 @@
+<?php
+/**
+ * Install tasks.
+ */
+
+/**
+ * Implements hook_enable().
+ */
+function stanford_help_enable() {
```

Comment:
Can you add a little more to this function description to what it is doing and why?